### PR TITLE
update translations/zh-CN.json

### PR DESCRIPTION
**Pull request by uos-ljtian**
update translations/zh-CN.json

### CODE REVIEW

**Comment by uos-ljtian** [comment]
Comparison of Changes in Line 591
Original:
image

result:
image

File: translations/zh-CN.json [hunk 1]
```diff
@@ -574,21 +574,21 @@
...
 	"One of 'yaml' or 'json'.": "'yaml'或'json'中的一个。",
-	"Only alphanumeric and dashes '-' are permitted. Minimum 1 character, starting with alphanumeric.": "",
+	"Only alphanumeric and dashes '-' are permitted. Minimum 1 character, starting with alphanumeric.": "仅允许字母数字和破折号'-'。最少1个字符，以字母数字开头。",
 	"Only alphanumeric and dashes '-' are permitted. Minimum 2 characters, starting with alphanumeric.": "仅允许字母数字和破折号'-'。最少2个字符，以字母数字开头。",
-	"Open the addons URL with https instead of http": "使用 https 替代 http 打开插件URL",
-	"Open the service URL with https instead of http (defaults to \"false\")": "",
-	"Opening Kubernetes service  {{.namespace_name}}/{{.service_name}} in default browser...": "",
+	"Open the addons URL with https instead of http": "使用 https 替代 http 打开插件的 URL",
+	"Open the service URL with https instead of http (defaults to \"false\")": "使用 https 替代 http 打开服务的 URL（默认为 \"false\"）。",
+	"Opening Kubernetes service  {{.namespace_name}}/{{.service_name}} in default browser...": "在默认浏览器中打开 Kubernetes 服务 {{.namespace_name}}/{{.service_name}}...",
```
Comment (liangyuanpeng) [inline translations/zh-CN.json:584]:
```suggestion
	"Opening Kubernetes service  {{.namespace_name}}/{{.service_name}} in default browser...": "正通过默认浏览器打开 Kubernetes 服务 {{.namespace_name}}/{{.service_name}}...",
```
与下一行翻译保持一致

File: translations/zh-CN.json [hunk 2]
```diff
@@ -574,21 +574,21 @@
...
+	"Options:      {{.options}}": "选项：{{.options}}",
+	"Output format. Accepted values: [json, yaml]": "输出格式。可接受的数值：[json, yaml]",
```
Comment (liangyuanpeng) [inline translations/zh-CN.json:590]:
```suggestion
	"Output format. Accepted values: [json, yaml]": "输出格式。可接受的值：[json, yaml]",
```

**Review by spowelljr — APPROVED** [review]

**Pull request merged**